src/sota_tools/deploy_test.cc

```diff
@@ -5,7 +5,9 @@
 #include "ostree_dir_repo.h"
 #include "ostree_http_repo.h"
 #include "ostree_ref.h"
+#include "server_credentials.h"
```

Comment:
Looks like we don't need this

Reply:
Probably not, since it is included in `deploy.h`.